Add docs and unit tests for dashboard logo (#278)
mckinsey/vizro: vizro-core/docs/pages/user_guides/dashboard.md

If supplied, the `title` of the [`Dashboard`][vizro.models.Dashboard] displays a heading at the top of every page.

## Adding a dashboard logo

> Vizro will automatically incorporate the dashboard logo in the top-left corner of each page if an image named 'logo' is present within the assets folder. The supported file extensions include ["apng", "avif", "gif", "jpeg", "jpg", "png", "svg", "webp"].

Reviewer: ```suggestion
Vizro will automatically incorporate the dashboard logo in the top-left corner of each page if an image named `logo.<extension>` is present within the assets folder. Image types of `apng`, `avif`, `gif`, `jpeg`, `jpg`, `png`, `svg`, and `webp` are supported.
```

Just to align it more with what we say on the meta images.

> 

Reviewer 2: I would probably just add a screenshot here with a dashboard title and logo in place. Just so they know where it will be positioned
